src/encoded/static/scss/encoded/modules/_genome_browser.scss

```diff
@@ -67,7 +67,7 @@
 // styles for bigBed files / annotation files
 .hpgv_track-annotation {
     --transcript-arrow: rgba(0, 0, 0, 1.0);
-    --transcript: #BDC5C0;
+    /* --transcript: #BDC5C0; */
```

Comment:
I think it's good to leave this as a comment so that there is documentation of how to set the transcript colors in CSS though generally I would say we should delete unused CSS.